source/features/releases-tab.tsx

```diff
@@ -20,7 +20,7 @@ const repoKey = `releases-count:${repoUrl}`;
 // Get as soon as possible, to have it ready before the first paint
 const cached = cache.get(repoKey);
 
-function updateReleasesCount() {
+function updateReleasesCount(): number | Promise<any> {
```

Comment:
Perhaps?

```suggestion
function updateReleasesCount(): number | Promise<number> {
```

Reply:
I tried this, but it means you'd have to `await` `cached`, e.g. `return Number(await cached)` because cached values are always strings from what I can tell, which works, but it seems odd to await after as well where it's called. I don't think there is a performance issue because if the Promise is already resolved it will return right away.

If you do a `get<T>(key)` like you mentioned below, it'd probably work. The only thing is it's a string coerced into a number versus `Number(...)`

Reply:
I'm not sure, I just tried `number | Promise<number>` locally and neither TS nor XO complain

Reply:
Ahh it doesn't complain because the cache returns `Promise<any>` which is why `Promise<number>` works even though it's a string when returned from the cache.